README.md

```diff
@@ -14,8 +14,59 @@ the packages individually, for example:
     make -C debs/xenial/archivematica
     make -C debs/xenial/archivematica-storage-service
 
-### Test packages
+#### Using parameters
 
+Most makefiles support parameters in the format of environment variables. They 
+may be a bit different between packages, but the most common are:
+    BRANCH
+    VERSION
+    RELEASE
+
+So in order to build an specific branch / version, this command can be used from
+the folder of the package we want to build:
+
+    make BRANCH=qa/1.x VERSION=1.7.0 RELEASE=rc5
+
+Have in mind that the Makefiles are a bit recursive, they will invoke docker, 
+mount the current folder, and run the deb-build target.
+
+#### Development
+
+Some makefiles have a "dev" target, that give you a shell inside of the docker 
```

Comment:
I think "inside of the docker used" should be "inside of the docker container used". Also, "to build the packages" sounds better to me than "to build packages".

Reply:
Done